PyPiDescription.rst

```diff
@@ -52,7 +52,7 @@ or for GPU instance:
 
 ::
 
-    pip install mxnet-cu90-mkl
+    pip install mxnet-cu90mkl
```

Comment:
Should we change to cu92mkl ?

Reply:
Ya, I think we can. No real reason for it not to be cu92mkl.